plugins/modules/user_role_workflow_manager.py

```diff
@@ -1442,6 +1444,10 @@ def create_user(self, user_params):
             if key not in user_params:
                 missing_keys.append(key)
 
+        if missing_keys:
+            error_message = "Mandatory parameter(s) {0} not present in the user details".format(", ".join(missing_keys))
```

Comment:
Adding single quotes around the parameter names can make the message more readable and help distinguish the parameters from the rest of the text.

`    error_message = "Mandatory parameter(s) '{0}' not present in the user details.".format(", ".join(missing_keys))
`
Also return error_message instead of error for more clarity..
`    return {"error_message": error_message}
`

Reply:
Updated the code accordingly.